### PR TITLE
Upgrade to latest react-native (0.44.x)

### PR DESCRIPTION
**Pull request by philikon**

### CODE REVIEW

File: android/app/src/main/java/com/rctwebrtcdemo/MainApplication.java [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 import android.app.Application;
-import android.util.Log;
 
 import com.facebook.react.ReactApplication;
-import com.facebook.react.ReactInstanceManager;
 import com.facebook.react.ReactNativeHost;
 import com.facebook.react.ReactPackage;
 import com.facebook.react.shell.MainReactPackage;
-import com.oney.WebRTCModule.WebRTCModulePackage;
+import com.facebook.soloader.SoLoader;
```
Comment (zxcpoiu) [inline android/app/src/main/java/com/rctwebrtcdemo/MainApplication.java]: oops, seems you killed `com.oney.WebRTCModule.WebRTCModulePackage` therefore `cannot find symbol`

**Review by zxcpoiu — CHANGES_REQUESTED** [review]
Hi @philikon long time no see

I don't have iOS device to test it now.
have you tested on iOS?

**Comment by philikon** [comment]
Yes, I have tested on iOS. Sorry, I did not test Android before submitting this PR. I'll fix the Android build.

**Comment by philikon** [comment]
Btw, have you seen https://github.com/oney/react-native-webrtc/issues/274? This app is affected too.

**Comment by philikon** [comment]
The latest commit appears to be a workaround for https://github.com/oney/react-native-webrtc/issues/274, but the underlying issue is not solved.

**Comment by zxcpoiu** [comment]
thanks @philikon 

merge this first.
we can continue the discussion in https://github.com/oney/react-native-webrtc/issues/274
I don't have video use case though but folks at jitsi might have some thoughts on it.

**Review by zxcpoiu — APPROVED** [review]

**Pull request merged**